utils/post_processing/backconvert_borealis.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2019 SuperDARN Canada, University of Saskatchewan
+# Author: Liam Graham
+
+#
+# backconvert_borealis.py
+# 2019-07-30
+# Command line tool for converting restructured and compressed
+# Borealis files back to their initial site format
+import deepdish as dd
+import numpy as np
+import os
+import sys
+import subprocess as sp
+import datetime
+import warnings
+
+
+def backconvert_pre_bfiq(data_record):
+	"""
+	Converts a restructured antennas iq file back to its
+	original site format
+	Args:
+		data_record (dict): An opened antennas_iq hdf5 file
```

Comment:
Can include a 'returns' field in the docstring for the ts_dict
Same for the backconvert_bfiq, and backconvert_rawacf